src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -817,13 +821,13 @@
     <Compile Include="Workarounds\DummyTypes\PermissionSet.cs" />
     <Compile Include="Workarounds\DummyTypes\RequestCachePolicy.cs" />
     <Compile Include="Workarounds\DummyTypes\SecurityManger.cs" />
-    <Compile Include="Workarounds\SerializationEntry.cs" />
-    <Compile Include="Workarounds\ICloneable.cs" />
+    <None Include="Workarounds\SerializationEntry.cs" />
```

Comment:
Will do. (**Note:** I'm going to do a cleanup for all the XML workarounds soon tracked by issue https://github.com/dotnet/corefx/issues/12000)